roles/provision_vcenter/templates/vcsa_deployment_config.json.j2

```diff
@@ -0,0 +1,30 @@
+{
+    "__version": "2.13.0",
+    "__comments": "Template to deploy a vCenter Server Appliance.",
+    "new_vcsa": {
+        {% filter indent(width=8, first=false) %}
+        {%- if provision_vcenter_vc_target_datacenter is defined %}
+        {%- include './templates/config_parts/auth_vc.j2' %},
+        {% else %}
+        {%- include './templates/config_parts/auth_esxi.j2' %},
+        {% endif %}
+        {% endfilter %}
+
+        {% filter indent(width=8, first=false) %}
+        {%- include './templates/config_parts/vm.j2' %},
+        {% endfilter %}
+        
+        "sso": {
+            {% if provision_vcenter_replication_partner_hostname is defined -%}
+            "first_instance": false,
+            "replication_partner_hostname": "{{ provision_vcenter_replication_partner_hostname }}",
+            "sso_port": 443,
+            {% endif -%}
+            "password": "{{ provision_vcenter_vm_password }}",
+            "domain_name": "vsphere.local"
```

Comment:
I don't understand those configurations, but is this right?

Reply:
yes, it is odd but that is what vmware expects. I think its because "sso" for vmware means domain specific login and the default domain for vcenter is vsphere.local. Once provisioned, a user could create other domains (I know it integrates with Windows/Active Directory) or create more users in the vsphere.local domain.